Core/Src/monitor.c

```diff
@@ -97,6 +97,14 @@ void vPedalsMonitor(void* pv_params)
 	// nertimer_t diff_timer;
 	// nertimer_t sc_timer;
 	// nertimer_t oc_timer;
+
+	/*diff_timer: timer for difference between pedal sensor values*/
+	nertimer_t diff_timer;
+	/*sc: short circuit tiimer*/
+	nertimer_t sc_timer;
+	/*oc: open circuit timer*/
+	nertimer_t oc_timer; 
```

Comment:
Seems like we use the same timer for brake and pedal faults, this will pose a problem since they are both referencing the same debounce

Reply:
i've added custom timers for the accelerator and brake sensors separately.